.changeset/old-bears-admire.md

```diff
@@ -0,0 +1,6 @@
+---
+"@primer/brand-primitives": patch
+"@primer/react-brand": patch
+---
+
+Remove the hover text-decoration transition in `InlineLink`, `Accordion`, `Timeline` and align `Prose` with the updated `InlineLink` styles
```

Comment:
Great, thanks @stamat.

On the off-chance that someone was relying on that downstream, let's provide a migration path:

```diff
- var(--brand-InlineLink-transition-timing)
+ var(--brand-animation-duration-fast)
```

Reply:
```suggestion
Removed a redundant hover text-decoration transition in `InlineLink`, `Accordion`, `Timeline` and align `Prose`.

Also removed the following css variable: `--brand-InlineLink-transition-timing`. If you were using this variable, please use `--brand-animation-duration-fast` instead.
```